lib/Ohmbrewer_Temperature_Sensor.cpp

```diff
@@ -2,42 +2,30 @@
 #include "Ohmbrewer_Screen.h"
 #include "Ohmbrewer_Onewire.h"
 
-/**
- * The last temperature read by the sensor. Currently returns in Celsius.
- * @returns A pointer to the Temperature object representing the last temperature reading
- */
-Ohmbrewer::Temperature* Ohmbrewer::TemperatureSensor::getTemp() const {
-    return _lastReading;
-}
-
-/**
- * The last time the temperature was read by the sensor
- * @returns The last temperature reading time
- */
-int Ohmbrewer::TemperatureSensor::getLastReadTime() const {
-    return _lastReadTime;
-}
 
 /**
- * Sets the last time the temperature was read by the sensor
- * @param lastReadTime The last temperature reading time
+ * Constructor
+ * @param id The Sprout ID to use for this piece of Equipment
+ * @param busPin The Digital Pin that the temp probes are attached to. NOTE: for ds18b20 should always be D0
  */
-const int Ohmbrewer::TemperatureSensor::setLastReadTime(const int lastReadTime) {
-    _lastReadTime = lastReadTime;
-    return 0;
+Ohmbrewer::TemperatureSensor::TemperatureSensor(int id, int busPin) : Ohmbrewer::Equipment(id) {
```

Comment:
Since we don't provide a Onewire constructor that picks a provided bus pin, we should drop the bus pin parameter to this constructor for now.